axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/InvoiceLineSupplychainService.java

```diff
@@ -226,7 +226,14 @@ public boolean checkTaxRequired(InvoiceLine invoiceLine, Integer packPriceSelect
         subLine.setIsSubLine(true);
         subLine.setPackPriceSelect(packPriceSelect);
         String description = null;
-        if (appAccountService.getAppInvoice().getIsEnabledProductDescriptionCopy()) {
+        if ((invoice.getOperationTypeSelect() == 3
```

Comment:
Please, don't use hard coded value.

Better to call the InvoiceToolService.isPurchase(Invoice invoice) method to know if it's a supplier or customer invoice.
https://github.com/axelor/axelor-business-suite/blob/master/axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceToolService.java#L71